src/libMessage/Messenger.cpp

```diff
@@ -3228,7 +3242,17 @@ bool Messenger::GetDSPowPacketSubmission(
     return false;
   }
 
-  for (const auto& powSubmission : result.dspowsubmissions()) {
+  ProtobufByteArrayToSerializable(result.pubkey(), pubKey);
+  Signature signature;
+  ProtobufByteArrayToSerializable(result.signature(), signature);
+  vector<unsigned char> tmp(result.data().ByteSize());
```

Comment:
use bytes instead of vector<unsigned char>

Reply:
Fixed.